spaces/cover/index.json

```
@@ -21,7 +21,8 @@
     "0xE24e48610D41819F9Ce16292DAD50be55A58C3df",
     "0x71864f87fE00Bf85612f13956F1A201B1C71CfC5",
     "0x9ba83F4BdBB1268B218337989f40293500333819",
-    "0xfe0842306A8548Ad327bCFC3f665877c45bEab60"
+    "0xfe0842306A8548Ad327bCFC3f665877c45bEab60",
+    "0xDd79dc5B781B14FF091686961ADc5d47e434f4B0",
```

Please delete this comma.
```suggestion
    "0xDd79dc5B781B14FF091686961ADc5d47e434f4B0"
```